evals/fixtures/review-quality/codebase/src/batch_processor.py

```
@@ -0,0 +1,374 @@
+import threading
+import sqlite3
+from datetime import datetime, timedelta
+from typing import Optional
+
+
+# --- Configuration ---
+
+DEFAULT_BATCH_SIZE = 100
+CLEANUP_THRESHOLD_DAYS = 90
+DB_PATH = "users.db"
+API_SECRET_KEY = "sk_live_09f8a7b6c5d4e3f2a1b0c9d8e7f6"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "API_SECRET_KEY"

(S105)